### ISSUE TITLE
latest release 1.2.2 not installing

### ISSUE THREAD

**Issue by SebastianBehrens** [wx-Yao/alfred-cheat#31]
Steps to reproduce (assuming cheat not yet installed)
1. Download Release 1.2.2
2. Open the file
3. Get the following message

<img width="305" alt="image" src="https://github.com/wayneyaoo/alfred-cheat/assets/51058351/571f4d8a-57bb-4368-9b5c-eeb91c085605">

Or is this just me?

**Comment by wayneyaoo** [comment]
Which macOS version is it? and is it Alfred.. 5? No a mac user anymore and lose track on the Alfred update.. maybe that's why.

**Comment by SebastianBehrens** [comment]
Ventura 13.5.2 and Yes, Alfred 5.1.4.

**Comment by mjs271** [comment]
Same issue on Sonoma 14.3 and Alfred 5.1.4

**Comment by odapg** [comment]
@SebastianBehrens @mjs271 Out of curiosity, do you encounter the same problem on my fork?

**Comment by SebastianBehrens** [comment]
@odapg No, I do not. Thank you!

**Comment by wayneyaoo** [comment]
interesting did you do anything to support Alred5 and the new OS? I didnt find similar commit. @odapg

**Comment by odapg** [comment]
@wayneyaoo No, I did nothing special to support Alfred5. My fork is mainly yours (including the corrections of @giovannicoppola), and an addition for the Quicklook view (it was my understanding that you preferred not to incorporate it in your version, to remain in the frame of your original idea, which I can perfectly understand). Since @SebastianBehrens does not meet the same problem with my release, I suspect that there is nothing wrong with your code, but that there was an issue when exporting the workflow to make the release.

**Comment by mjs271** [comment]
@odapg, that did the trick--much appreciated! 🙌

**Comment by plphlpp** [comment]
fyi : same issue here (macOS 10.15.7. Alfred 5.1.4) : unable to import,invalid/corrupted file
@odapg your fork 87cbf7b works flawlessly. thanks
